static/src/javascripts/projects/common/modules/experiments/tests/contributions-embed.js

```diff
@@ -0,0 +1,124 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-embed.html',
+    'common/utils/robust',
+    'text!common/views/giraffe-message.html',
+    'inlineSvg!svgs/icon/arrow-right'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEmbed,
+             robust,
+             giraffeMessage,
+             arrowRight
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsEmbed20160823';
+        this.start = '2016-08-23';
+        this.expiry = '2016-08-30';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test contributions embed with amount picker.';
+        this.showForSensitive = false;
+        this.audience = 0.10;
+        this.audienceOffset = 0.13;
+        this.successMeasure = 'Impressions to number of contributions';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'The interactive embed performs 3x better than the control';
+        this.canRun = function () {
+            var pageObj = window.guardian.config.page;
+            return !(pageObj.isSensitive || pageObj.isLiveBlog || pageObj.isFront || pageObj.isAdvertisementFeature) && pageObj.edition === 'UK';
+        };
+
+
+        var writer = function (component) {
+
+            return fastdom.write(function () {
+                var a = $('.submeta');
+                component.insertBefore(a);
+                mediator.emit('contributions-embed:insert');
```

Comment:
I think you can pass the component as the second argument:

`mediator.emit('contributions-embed:insert', component);`

and then you would just be able to do:

```
mediator.on('contributions-embed:insert', function (el) {
   require([el.getAttribute('data-interactive')], function 
```

below.